beacon-chain/powchain/service.go

```diff
@@ -297,6 +298,27 @@ func (w *Web3Service) handleDelayTicker() {
 	}
 }
 
+func (w *Web3Service) handleSkippedHeaders(ctx context.Context, header *gethTypes.Header) {
+	headerHeight := header.Number.Uint64()
+	currentHeight := w.blockHeight.Uint64()
+
+	if headerHeight > currentHeight+1 {
```

Comment:
Do we need this if condition. The for loop should just take care of this right?

Reply:
yeah the condition is redundant in this case, removed it